javapoet/src/main/java/com/palantir/javapoet/TypeSpec.java

```diff
@@ -492,40 +520,53 @@ public String toString() {
 
     @SuppressWarnings("ImmutableEnumChecker")
     public enum Kind {
-        CLASS(Collections.emptySet(), Collections.emptySet(), Collections.emptySet(), Collections.emptySet()),
+        CLASS(Collections.emptySet(), Collections.emptySet(), Collections.emptySet(), Collections.emptySet(), "class"),
```

Comment:
Remove the `toString` implementation here - we no longer use it. This removes the last remaining use of `toLowerCase`.